src/main/java/lv/acodemy/Homework/VariablesDataTypes1.java

```diff
@@ -0,0 +1,58 @@
+package lv.acodemy.Homework;
+
+import java.sql.SQLOutput;
+import java.util.Arrays;
+
+public class VariablesDataTypes1 {
+    public static void main(String[] args) {
+        byte myCurrentAge = 29;
+        byte workYears = 6;
+        byte mortgageYears = 20;
+        short monthlySalary = 1300;
+        short currentYear = 2023;
+        short BirthYear = 1993;
```

Comment:
incorrect naming

Reply:
BirthYear should start with lowercase letter. Is there something else incorrect with those lines?